lib/active_utils/connection.rb

```diff
@@ -45,7 +48,7 @@ def initialize(endpoint)
       @max_retries  = MAX_RETRIES
       @ignore_http_status = false
       @ssl_version = nil
-      @proxy_address = nil
+      @proxy_address = PROXY_ADDRESS
```

Comment:
Rationale behind that was to be able to use EVN['http_proxy'] as well as cert_file for it(which currently a constant in `Connection`) without the need to put that into the shipped code. This way any testing done with proxy use wouldn't have any effect on the code. 

That would also align with how `net/http` behaves by default. When `http_proxy` is set in the ENV it will use it. If not then it won't. 

Default was `nil` so if the default will be something else to let `net/http` do it's job then it should be changeable in the same way as you mentioned, right?

Reply:
While I understand that the users of this lib might prefer explicitly configuring using the environmental variable, I think it _should_ be the default: in general, all HTTP-aware apps should honour `http_proxy` and to the best of my knowledge all Ruby HTTP libraries do, so having that variable set and it _not_ being a default would be surprising to me. 👍 to this implementation.